bhx5chain/CHANGELOG.md

```diff
@@ -17,6 +30,5 @@ Versioning](https://doc.rust-lang.org/cargo/reference/semver.html).
 - README.md describing the crate.
 - Initial version of the `bhx5chain` crate.
 
-
```

Comment:
I've deliberately put two empty lines in order to have a more clear separation of the footnote and the content under the header of a version.